public/index.html

```diff
@@ -1,26 +1,42 @@
-<!DOCTYPE HTML>
+﻿<!DOCTYPE HTML>
 <!--
 	Story by HTML5 UP
 	html5up.net | @ajlkn
 	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
 -->
+
 <html>
 	<head>
+        <meta name="description" content="Hack Washington is the first ever collegiate hackathon to be hosted in the Seattle Space Needle.
+              Hackers from all across the US and Canada are invited to participate in the Hack Washington hackathon." />
+        <meta name="Hacker Info" content="Hackers will be hacking inside the Space Needle, an exciting aspect of Hack washington."/>
+        <meta name="Hack Washington sponsorship info" content="Some info about Hack Washington dates and venue"/>
+        <meta name="Hack Washington FAQ" content="FAQs regarding Hack Washington @ The Seattle Space Needle."/>
+        <meta name="Hack Washington Info" content="Hack Washington dates and venue, Seattle Space Needle"/>
+        <meta name="Hack Washington Sponsor Logos" content="The logos of and info about each Hack Washington @ The Seattle Space Needle sponsor"/>
+
 		<title>Hack//WA</title>
     	<link rel="icon" type="image/png" href="http://www.kavehkhorr.am/hackwashington/website_stuf/favicon.jpg" sizes="50x50" />
 		<meta charset="utf-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
 		<link rel="stylesheet" href="assets/css/main.css" />
 		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
 	</head>
+<nav>

```

Comment:
Is this a visual change? If so, what's the visual change here?

Reply:
I added a navigation bar to the top of the page that allows visitors to jump to each section of the site.

Reply:
What does that visual change look like?

Reply:
Attach a screenshot, please.

Reply:
![hackwanavbarss](https://user-images.githubusercontent.com/19559974/45922691-570b7b00-be88-11e8-8dff-02c216b9503f.PNG)

Reply:
The bar doesn't follow the user down the page. Meaning that the user can scroll past it.